### PR TITLE
Update cl.xml for new vendor IDs in vk.xml

### PR DESCRIPTION
**Pull request by franz**
This depends on PR to [add](https://github.com/KhronosGroup/Vulkan-Headers/pull/169) PoCL vendor ID in Vulkan headers.

### CODE REVIEW

File: xml/cl.xml
Comment (EwanC) [inline xml/cl.xml:1091]: We're using this vendor ID, looks like it's getting removed. Should PoCL be getting added rather than removing Codeplay?

**Review by EwanC — CHANGES_REQUESTED** [review]

**Review by bashbaug — CHANGES_REQUESTED** [review]
I think you'll want a Vulkan-Docs PR first, to reserve the vendor ID enum.  See, for example, https://github.com/KhronosGroup/Vulkan-Docs/pull/1168, which reserved the Codeplay enum.

Note that this should be a Vulkan-Docs PR and not a Vulkan-Headers PR.

We should merge this PR after the Vulkan-Docs PR is merged.  Thanks!

File: xml/cl.xml
Comment (EwanC) [reply xml/cl.xml:1091]: Looks good now, thanks

**Review by alycm — COMMENTED** [review]
LGTM, needs the VK XML PR to merge first to reserve the value: https://github.com/KhronosGroup/Vulkan-Docs/pull/1411

Both PRs just need the CLA to be agreed.

**Review by EwanC — APPROVED** [review]
Accompanying https://github.com/KhronosGroup/Vulkan-Docs/pull/1411 is merged now

**Review by bashbaug — APPROVED** [review]
LGTM

**Pull request merged**
